### PR TITLE
docs/developer: corrected java version 8>11 as mentioned here: https:…

### PR DESCRIPTION
**Pull request by metzenseifner**
…//docs.opencast.org/r/10.x/admin/#upgrade/#upgrade-java

### Your pull request should…

* [ ] have a concise title
* [ ] [close an accompanying issue](https://help.github.com/en/articles/closing-issues-using-keywords) if one exists
* [ ] [be against the correct branch](https://docs.opencast.org/develop/developer/development-process#acceptance-criteria-for-patches-in-different-versions)
* [ ] include migration scripts and documentation, if appropriate
* [ ] pass automated tests
* [ ] have a clean commit history
* [ ] [have proper commit messages (title and body) for all commits](https://medium.com/@steveamaza/e028865e5791)

### CODE REVIEW

**Comment by metzenseifner** [comment]
Note that I changed the commit to have a "title" and a "body" *after* approval was reviewed by the bot (and failed due to a missing ICLA Agreement)

**Review by mtneug — APPROVED** [review]

**Review by lkiesow — APPROVED** [review]
Looks good, and I've seen that Apereo has received an ICLA. Thanks for that.

**Pull request merged**
